src/graphnet/data/extractors/icecube/i3truthextractor.py

```diff
@@ -13,7 +13,7 @@
 from graphnet.utilities.imports import has_icecube_package
 
 if has_icecube_package() or TYPE_CHECKING:
-    from icecube import (
+    from icecube import (  # noqa: F401
```

Comment:
I needed to add this because LeptonInjector isn't used directly, but it needs to be imported for icetray to recognize the "EventProperties" frame object type.